src/lib/y2partitioner/actions/controllers/lvm_vg.rb

```diff
@@ -419,14 +419,13 @@ def used?(device)
           true
         end
 
-        # Checks whether a device is mounted
+        # Checks whether a device has mount point
         #
         # @return [Boolean] true if it is mounted; false otherwise
         def mounted?(device)
           return false unless formatted?(device)
 
-          mount_point = device.filesystem.mount_point
-          mount_point && !mount_point.empty?
+          !device.filesystem.mount_point.nil?
```

Comment:
Is it possible that we have a mount point, but that mount point doesn't have a path or the path is empty? What would happen in that case?